images/build/Dockerfiles/Dockerfile

```diff
@@ -235,12 +211,6 @@ RUN cp ${IMAGES_DIR}/build/dotNetCoreAll.Readme.md /opt/dotnet/Readme.md
 # Copy NodeJs, NPM and Yarn related content
 COPY --from=node-install /opt /opt
 
-# Install PHP pre-reqs
-RUN ${IMAGES_DIR}/build/php/prereqs/installPrereqs.sh
-
-# Oryx depends on the run script generators for most of its
-# `IProgrammingPlatform.GenerateBashRunScript()` implementations
-COPY --from=startupScriptGens /opt/startupcmdgen/ /opt/startupcmdgen/
```

Comment:
So wondering why we're removing all php prereqs files, we don't need them anymore?

Reply:
Do we want to change build image tests accordingly or it won't affect those tests

Reply:
ans1. yes we may have some failing build images. idea is to find them and make sure they are taken care of by either removing them (don't need anymore) or fixing them.
ans2. so … we are now dynamically installing  php binaries in the build image .. so we don't need to create php build bases anymore .. that's why we can get rid of these files.
.

Reply:
Got it, thanks. Let's make sure all those tests will pass.